Sources/EpoxyCollectionView/CollectionView/CollectionView.swift

```diff
@@ -179,10 +188,10 @@ open class CollectionView: UICollectionView {
         let items: [CollectionViewVisibilityMetadata.Item] = visibleItems.compactMap { indexPath in
```

Comment:
I was curious if we could receive an index OOB assertion from line 177 if we were in the middle of a batch update and a section was being removed but still visible (similar to our issue seen with cells) since that section would only be in the old data source. I wasn't able to reproduce in the Epoxy sample project though. I wonder if this issue generally only showed up on the PDP due to some specific conditions or timings.